src/common/dependency-manager/local-manager.ts

```diff
@@ -109,15 +109,15 @@ export default class LocalDependencyManager extends DependencyManager {
     if (spec.metadata.file?.path && this.environment === 'local') {
```

Comment:
I think I want it to only merge if debug is true - the environment could be anything. Let me know if this doesn't seem right.

```ts
if (spec.metadata.file?.path && debug) {
```

Reply:
That makes sense.